src/redux/backend-session.spec.ts

```diff
@@ -205,4 +206,32 @@ describe("epics", () => {
       store.getActions().find(action => action.type === startSession.type)
     ).toBeDefined();
   });
+
+  describe("selectImageTag", () => {
+
+    test("should return the python image tag based on version chosen", () => {
+      const language = "python";
+      const version5 = "3.5";
+
+      expect(selectImageTag(language, version5) === "dcl-python-3.5:latest");
```

Comment:
In order to provide better output in case of failure, I suggest:

```js
expect(selectImageTag(language, version5)).toEqual('dcl-python-3.5:latest');
```

Reply:
Ok cool, thanks for that!